tests/conftest.py

```diff
@@ -229,6 +229,14 @@ def duthost(ansible_adhoc, testbed, request):
     if stop_ssh_timeout is not None:
         enable_ssh_timout(duthost)
 
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
I think in the future, the test case that is altering the critical services should invoke a Fixture to reset it back to default ones, what do you think?

Reply:
I don't think we necessarily need a fixture since there is a "reset to default" method in duthost. I agree that it makes sense that the caller should handle restoring the default, I'm just worried that if someone forgets then it might mess with the rest of the test cases.

Reply:
I'll go ahead with this change so it's not blocking your PR and we can re-visit this later. I think it only really affects the VRF tests at this point so we can discuss some more and open another PR later if we feel it is appropriate.